### ISSUE TITLE
<div> with id="MathJax_Message" not included in ARIA landmark

### ISSUE THREAD

**Issue by froggleston** [mathjax/MathJax#3546]
**Is your feature request related to a problem? Please describe.**

Currently at least one of the inserted divs by mathjax are not conformant to WCAG, as the `MathJax_Message` div has no encompassing landmark element, being a direct descendent to the `<body>`.

**Describe the solution you'd like**

Either wrap the `MathJax_Message` div within a landmark element, or change the div element itself to a landmark.

**Additional context**

[Issue showing an example](https://github.com/carpentries/varnish/issues/200)

**Comment by dpvc** [comment]
The `MathJax_Message` div is only used in MathJax v2, which is now something like 6 years out of date.  Version 3 was released in 2019, and the last v2 release was in 2020.  The current version is 4.1.1.  Neither v3. nor v4 include the `MathJax_Message` feature.

It is unlikely that there will be a v2 update to address this, especially since the contents of the message are transient (only present for a short time while a file is loading, in general), and so not something an assistive user is likely to try to navigate to.

In any case, this is no longer an issue for current versions of MathJax, only for the legacy version 2.

**Comment by froggleston** [comment]
Hoo boy, I didn't notice that the version we were using was so out of date! Apologies, I'll update and verify my end! Sorry for the noise 👍

**Comment by dpvc** [comment]
Note that the API changed significantly with v3, so if your code calls the v2 API, that may require some rewriting to move to v3/v4.  See the [documentation](https://docs.mathjax.org/en/latest/upgrading/v2.html) on converting from v2 to v3 (still relevant to v4).

**Comment by dpvc** [comment]
I just checked the varnish repository, and it looks like you don't use the API, so conversion should be straight-forward.  The only change it looks like you will need to do is update configuration.  The configuration mechanism is different in v3, and the configuration options are different, so you will need to change your current v2 configuration to a v3/v4 one.  Fortunately, there is a [conversion tool](https://mathjax.github.io/MathJax-demos-web/convert-configuration/convert-configuration.html) to help you do that.

**Comment by froggleston** [comment]
@dpvc That's kind of you to check out our repo and provide this information - it's much appreciated! I'll take a look at this next week 👍

**Comment by dpvc** [comment]
No problem.  Good luck, and let me know if you have problems.
